modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the type of storage provided by your selected storage class. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.
+
+For example:
+
+* When using Ceph RBD, your access mode is automatically set to `ReadWriteMany`, which improves efficiency during Live Migration, among other performance benefits.
```

Comment:
Which enables migration (migration won't work with any other access mode)

Reply:
Block mode is chosen for performance benefits.

Reply:
Reworked this section a bit:

When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the type of storage provided by your selected storage class. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.

For example:

* When using Ceph RBD, your `accessModes` is automatically set to `ReadWriteMany`, which enables Live Migration. `volumeMode` is set to `Block` to maximize performance.
* When not using block allocation, filesystem overhead is automatically calculated for you, if required.

In the following YAML, using the *storage* API allocates a data volume with a storage capacity of two gigabytes, as in the previous *pvc* API allocation example. By using the *storage* API, you are guaranteed two gigabytes of usable space. It is not required to specify specify `accessModes` or `volumeMode`.  These values are optimized and calculated for you by the CDI, based on the type of storage or the defaults that you define in your storage profile. If you want to provide custom values, they override the system-calculated values.